src/simsopt/field/coil.py

```diff
@@ -185,3 +186,108 @@ def coils_via_symmetries(curves, currents, nfp, stellsym):
     currents = apply_symmetries_to_currents(currents, nfp, stellsym)
     coils = [Coil(curv, curr) for (curv, curr) in zip(curves, currents)]
     return coils
+
+
+def coils_to_makegrid(filename, curves, currents, groups=None, nfp=1, stellsym=False):
+    """
+    Export a list of Curve objects together with currents in MAKEGRID format, so they can 
+    be used by MAKEGRID and FOCUS. The format is introduced at
+    https://princetonuniversity.github.io/STELLOPT/MAKEGRID
+
+    Args:
+        filename: Name of the file to write.
+        curves: A python list of Curve objects.
+        currents: Coil current of each curve.
+        groups: Coil current group. Coils in the same group will be assembled together. Defaults to None.
+        nfp: The number of field periodicity. Defaults to 1.
+        stellsym: Whether or not following stellarator symmetry. Defaults to False.
+    """
+
+    assert len(curves) == len(currents)
+    assert isinstance(curves, CurveXYZFourier)
```

Comment:
I don't think we need this here. You're just exporting XYZ coordinates, that works for every choice of discretisation.